lib/retrieve_title.rb

```diff
@@ -46,6 +46,7 @@ def self.max_chunk_size(uri)
     # but these are large sites.
     return 500 if uri.host =~ /amazon\.(com|ca|co\.uk|es|fr|de|it|com\.au|com\.br|cn|in|co\.jp|com\.mx)$/
     return 300 if uri.host =~ /youtube\.com$/ || uri.host =~ /youtu.be/
+    return 50 if uri.host =~ /github\.com$/
```

Comment:
I think the regex should be `/(^|\.)github.com/` because otherwise `sfsfsafsdfgithub.com` would get the same treatment as `github.com`?

Reply:
This is a good point and I think it affects the youtube and amazon regexp above.

Reply:
(nit: the comment above mentions just "Amazon and YouTube" so it would be better to remove those mentions as this list of sites expands 😃)